addon/model.js

```diff
@@ -433,20 +545,28 @@ export default class MegamorphicModel extends EmberObject {
 
     const isDirty = recordData.hasDirtyAttr();
```

Comment:
What is triggering a change notification on `isDirty` in the custom model class case?